### PR TITLE
informative description

### PR DESCRIPTION
**Pull request by monad0**

### CODE REVIEW

**Comment by kkthompson** [comment]
I don't think this is the right kind of text for a marketing-style site. While all "correct", it doesn't mean much to most of the people visiting the site. Maybe it goes in a paragraph under the tool, but definitely not the headline.

**Comment by monad0** [comment]
A rather ironic argument against illuminating terms of the very tool expected to convey meaning.

File: resources/distance-vs-effort-visualizer.html [hunk 1]
```diff
@@ -113,7 +113,7 @@
           <h1 class="display-xl-bold fill-gradient">Distance vs Effort Visualizer</h1>
         </div>
         <div class="wrapper-text width-small">
-          <h2 class="text-l-medium large">Interactively visualize how varying Distance (quality) and Effort (encoding effort) affect image appearance with the official JPEG XL reference encoder.</h2>
+          <h2 class="text-l-medium large">Interactively visualize how varying target distance and encoder effort affect image appearance with the official JPEG XL reference encoder.</h2>
```
Comment (jonsneyers) [inline resources/distance-vs-effort-visualizer.html:116]:
```suggestion
          <h2 class="text-l-medium large">Interactively visualize how varying the visual distance target (quality) and encoder effort affect image appearance with the official JPEG XL reference encoder.</h2>
```

File: resources/distance-vs-effort-visualizer.html [hunk 2]
```diff
@@ -152,6 +152,7 @@ <h2 class="text-l-medium large">Interactively visualize how varying Distance (qu
             </div>
           </div>
         </div>
+        <p>Distance measures visual distortion, based on the psychovisually robust butteraugli metric. Effort moderates compute allowed for achieving accurate distance and optimizing information density.</p>
```
Comment (jonsneyers) [inline resources/distance-vs-effort-visualizer.html:155]:
```suggestion
        <p>Distance measures visual distortion, based on the psychovisually robust butteraugli metric. Effort moderates computing power (cpu and memory) allowed for achieving accurate distance and optimizing compression density.</p>
```

**Comment by jonsneyers** [comment]
I agree with @kkthompson that putting everything in the headline makes the headline too long and scary. Putting some more information in a paragraph below is useful though.

I proposed some tweaks to the formulation.

**Pull request merged**